src/firebase/firestore/query/index.js

```diff
@@ -21,7 +21,7 @@ export default class Query {
 
     this._operations.forEach((operation) => {
       if (operation.type === 'orderBy') {
-        data.__doc__ = orderBy(data.__doc__, operation.param.key, operation.param.sorting);
+        data.__doc__ = orderBy(data.__doc__ || {}, operation.param.key, operation.param.sorting);
```

Comment:
Thanks for working on this. I haven't checked but looking back at my codes I think the issue will arise on all filters (`orderBy`, `startAt`, `endAt`, etc.)

Instead of this fix for `orderBy`, I think we should convert line 20 to:

```javascript
const data = Object.assign({ __doc__: {} }, this._data);
```

Is it okay for you to try it out? 😄

Reply:
Yep no problem :)

Reply:
It work ! All tests pass